js/common.js

```diff
@@ -0,0 +1,46 @@
+/* 에러 메시지 처리 */
+function wrongInput(input, span, validateFunction) {
+  const errorMessage = validateFunction(input);
+  if (errorMessage) {
+    input.classList.add("warning");
+    input.classList.remove("done");
+    span.textContent = errorMessage;
+  } else {
+    input.classList.remove("warning");
+    input.classList.add("done");
+    span.textContent = "";
+  }
+  input.parentElement.appendChild(span);
+  if (span.textContent === "") {
+    input.parentElement.removeChild(span);
+  }
+}
+
+/* 눈모양 버튼 클릭시 비밀번호 보이기 or 가리기 */
+function togglePasswordVisibility(button) {
+  const passwordInput = button.previousElementSibling;
```

Comment:
💊 제안
이렇게 접근하시는 것도 나쁘지 않지만 이는 HTML 구조와 긴밀하게 연관되게 되므로,
HTML에서 위치만 변경되어도 작동되지 않을 위험이 커집니다~
가능하면 부모요소로 접근해 해당 요소 내에서 재탐색을 해보세요.

```suggestion
  const passwordInput = button.parentElement.querySelector("input");
```